data/map systems.txt

```diff
@@ -11911,6 +12592,77 @@ system Eltanin
 		distance 2165.97
 		period 1900.78
 
+system Embe'kor
+	pos 72.444444 178.77778
+	government Korath
+	attributes korath warm
+	asteroids "small rock" 49 3.5672
+	asteroids "medium rock" 4 3.2032
+	asteroids "large rock" 46 3.458
+	asteroids "small metal" 2 3.458
+	asteroids "medium metal" 12 2.5844
+	asteroids "large metal" 9 1.9656
+	minables tungsten 5 5.213319
+	haze _menu/haze-orange
+	fleet "Korath Raid" 1600
```

Comment:
```suggestion
	fleet "Korath Raid" 1600
	fleet "Korath Ember Waste Raid" 10000
	fleet "Korath Rescue" 30000
```
This system should have the same Korath fleets found in Parca, since this is the only possible route they could have taken to get there.